SlevomatCodingStandard/Helpers/NamespaceHelper.php

```diff
@@ -45,6 +45,15 @@ public static function isFullyQualifiedName(string $typeName): bool
 		return StringHelper::startsWith($typeName, self::NAMESPACE_SEPARATOR);
 	}
 
+	public static function getFullyQualifiedTypeName(string $typeName): string
```

Comment:
Please move this method and test to separate commit.

Reply:
Not sure I understand. I thought I should squash everything into 1 commit. You ask to separate it. Does it mean 2 commits in this PR, or 2 separate PRs?

Reply:
This change can be in separate commit in this PR, so in this PR will be 2 commit. Changes in `NamespaceHelper` and the new sniff.

Reply:
Split into 2 commits